app/views/orders/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
+
+<h1>My bought rewards</h1>
+
+<table class="table">
+  <thead class="table-dark">
+    <tr>
+      <th>Title</th>
+      <th>Description</th>
+      <th>Purchase time</th>
+      <th>Price</th>
+
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @orders.each do |order| %>
+      <tr>
+      <% if current_employee == order.employee %>
```

Comment:
this is quite a lot of comparisons, what if there are 10_000_000 of orders in the db, but only 1 belongs to current user? Consider filtering out only orders belonging to the current user in controller - something like `current_user.orders`

Reply:
done :)